modules/react/common/lib/utils/useModalityType.ts

```diff
@@ -2,6 +2,22 @@ import React from 'react';
 
 type Modality = 'mouse' | 'touch' | 'pen';
 
+function safeLocalStorageGet(key: string): string | null {
+  try {
+    return localStorage.get(key);
+  } catch (_) {
+    return '';
```

Comment:
Not sure which is better, but what do you think about `return null;` here?

Reply:
`null` would be better, but the effect is the same. `''` and `null` are both falsey

Reply:
Ah, for the usage. I get you. Thanks for changing it. Just in case we refactor it later to be used more.